package.json

```
@@ -100,6 +100,7 @@
       "embark-dapp-template-demo/bootstrap",
       "embark-dapp-test-app/embark-dapp-test-service",
       "embark-dapp-test-app/zeppelin-solidity",
+      "teller-test-contracts/tabookey-gasless",
```

The left-hand side needs to be the name of the package, not the name of the directory. See also my other comment about maybe switching the name to `embark-dapp-test-teller-contracts`.

```suggestion
      "embark-dapp-teller-test-contracts/tabookey-gasless",
```